masterlist.yaml

```
@@ -8820,6 +8820,19 @@ plugins:
         util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 773
 
+  - name: 'WoodworkersWhim.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/64507'
+        name: 'Nexus Mods'
+    group: *lateChangesGroup
+    clean:
+      # Normal Plugin
+      - crc: 0x24E8B4EA
+        util: '[SSEEdit v4.0.1](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
```

Cleaning data should be submitted using either the latest stable or beta release of xEdit, with the beta release being preferred. QAC reports should also be left unedited, especially since, in this case, Markdown isn't supported for clean cleaning data (only dirty):

```suggestion
      - crc: 0x24E8B4EA
        util: 'SSEEdit v4.0.4b'
```

---

My bad on that. I copy-pasted from the one above to get the link, but it looks like I forgot to check the versioning. And it would seem that my efforts were in vain anyway.